### PR TITLE
ClimateMappedAfrica Default Variables

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description

This PR setups some default environment variables on ClimateMappedAfrica.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by VinneyJ — APPROVED** [review]

**Comment by kilemensi** [comment]
Can you may be describe what issue is this PR fixing? For example, I don't think `NEXT_PUBLIC_SEO_DISABLED` is even supported in the `climatemappedafrica` app.

**Comment by kilemensi** [comment]
Isn't that same issue fixed in https://github.com/CodeForAfrica/ui/pull/977 @kelvinkipruto ?

**Comment by kelvinkipruto** [comment]
@kilemensi Yes, it can fix it. I hadn't looked at your PR before working on this.

File: apps/climatemappedafrica/.env [hunk 1]
```diff
@@ -1,3 +1,7 @@
+NEXT_PUBLIC_IMAGE_UNOPTIMIZED=true
+NEXT_PUBLIC_SEO_DISABLED=true
+PAYLOAD_PUBLIC_DEFAULT_LOCALE=en
+PAYLOAD_PUBLIC_LOCALES="en"
```
Comment (koechkevin) [inline apps/climatemappedafrica/.env:4]: Are they needed at buildtime?

Comment (kelvinkipruto) [reply apps/climatemappedafrica/.env:4]: looks like we do not need them

**Comment by kelvinkipruto** [comment]
This PR is not needed as the issues will be fixed in #977

**Pull request closed without merging**